docs/ref/contrib/postgres/operations.txt

```diff
@@ -59,3 +59,35 @@ the ``django.contrib.postgres.operations`` module.
 .. class:: UnaccentExtension()
 
     Installs the ``unaccent`` extension.
+
+``Create extension using migrations``
+=====================================
+
+You can create a PostgreSQL Extension in your database using a migration file.
+The following example will cover creating of hstore extension,
```

Comment:
Spelling: "example will cover creating of hstore extension" => "example covers creating an hstore extension"